flink-libraries/flink-table/pom.xml

```diff
@@ -140,6 +140,20 @@ under the License.
 			<version>${project.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-streaming-java_2.10</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-runtime_2.10</artifactId>
+			<version>${project.version}</version>
+			<scope>test</scope>
+			<type>test-jar</type>
+		</dependency>
```

Comment:
Yes, the dependency is required for the test harness class which allows to control processing time. This is needed to properly test processing time operators that have a semantic that depends on the time (so not only sorting by time which is implicitly true for processing time). The processing time OVER RANGE window groups data by processing time. If we want to test this without manual timing (which is not good unit test practice) we need the test harness. There will be more operators in the future that require this dependency.